README.md

```diff
@@ -4,6 +4,14 @@ Stryker evaluator action for Tryber projects
 
 This action evaluate Tryber projects with [Stryker](https://www.npmjs.com/package/stryker) library.
 
+## Inputs
+
+- `pr_author_username`
```

Comment:
Seria interessante atualizar em como consumir a action com esse novo input obrigatório, nas seções [Usage example](https://github.com/betrybe/stryker-evaluator-action#usage-example) e [`How to get result output`](https://github.com/betrybe/stryker-evaluator-action#how-to-get-result-output), não?